plugin/hover.py

```diff
@@ -409,3 +409,10 @@ def _update_views_async(self, enable: bool) -> None:
                         session_view.view.settings().set(SHOW_DEFINITIONS_KEY, False)
                     else:
                         session_view.reset_show_definitions()
+
+
+class LspCopyTextCommand(sublime_plugin.TextCommand):
```

Comment:
Should this be a WindowCommand, so that it could also be used in a HtmlSheet?
In that case I would also use `self.window.status_message` instead of `sublime.status_message`

Reply:
Done [226e125](https://github.com/sublimelsp/LSP/pull/2618/commits/226e1256c07e3a9b01c3a162077fc6850d9b41b4)